geometry_msgs/PoseArray checkpoints
---
uint32 read_status
